Library/Homebrew/cmd/bundle.rb

```diff
@@ -273,6 +260,28 @@ def run
             require "bundle/commands/remove"
             Homebrew::Bundle::Commands::Remove.run(*named_args, type: selected_types.first, global:, file:)
           end
+        when *BUNDLE_EXEC_COMMANDS
```

Comment:
Would be nice to not move this so the diff is easier to follow?

Reply:
Was moved by `brew style --fix` 😅

Reply:
@carlocab oh no! will review harder.